srcpkgs/swirc/template

```diff
@@ -15,17 +17,15 @@ license="BSD-3-Clause, ISC, MIT"
 homepage="https://www.nifty-networks.net/swirc"
 changelog="https://raw.githubusercontent.com/uhlin/swirc/master/CHANGELOG.md"
 distfiles="https://www.nifty-networks.net/swirc/releases/${pkgname}-${version}.tgz"
-checksum="d37b24bb5ede44e34a1fcbdfe70cbd94b0f2bec5cc69c76ff73554b5544b0418"
+checksum="c9fcfbec47e2aaebf5a5d714a41eaa8f03b063d172a615b465d5f543020ca9b0"
 
 build_options="notify"
-
-post_extract() {
-	vsed -i "${wrksrc}/src/events/privmsg.cpp" -e "s,/usr/local/share/swirc,/usr/share/swirc,"
-	vsed -i "${wrksrc}/src/main.cpp" -e "s,/usr/local/share/locale,/usr/share/locale,"
-}
+build_options_default="notify"
 
 post_configure() {
-	local _file="options.mk"
+	local _file
+
+	_file="options.mk"
```

Comment:
this change isn't necessary, it doesn't do anything different

Reply:
True :)